src/test/scala/com/atomist/rug/function/github/GitHubFunctionTest.scala

```diff
@@ -18,15 +17,15 @@ abstract class GitHubFunctionTest(val oAuthToken: String, val apiUrl: String = "
 
   import TestConstants._
 
-  protected val ghs = GitHubServices(oAuthToken, apiUrl)
+  protected val ghs = GitHubServices(oAuthToken, Option(apiUrl))
 
   override protected def afterAll(): Unit = cleanUp()
 
   /**
     * Return a temporary repository callers can use.
     */
   def newTemporaryRepo(autoInit: Boolean = false) =
-    ghs.createRepository(getRepoName, TestOrg, "temporary test repository", true, true, autoInit)
+    ghs createRepository(getRepoName, TestOrg, "temporary test repository", privateFlag = true, autoInit = autoInit)
```

Comment:
the argument is for `issues` which is true by default if not sent to GitHub. All repos are created with issues enabled for the tests